src/OrbitGl/GlCanvas.cpp

```diff
@@ -43,8 +43,8 @@ float GlCanvas::kZValueSlider = 0.89f;
 float GlCanvas::kZOffsetMovingTack = 0.1f;
 float GlCanvas::kZOffsetPinnedTrack = 0.2f;
 
-// Max Number of layers: 16 original, 4 for moving track, 4 for pinned Track, 4 epsilon in Slider
-unsigned GlCanvas::kMaxNumberRealZLayers = 16 + 4 + 4 + 4;
+// Max Number of layers: 16 original, 5 for moving track, 5 for pinned Track, 4 epsilon in Slider
+unsigned GlCanvas::kMaxNumberRealZLayers = 16 + 5 + 5 + 4;
```

Comment:
Nit: Make these named constants, then you can get rid of some of the comment. 

constexpr unsigned kNumberOriginalLayers = 16;
constexpr unsigned kNumberLayersForMovingTracks = 5;
constexpr unsigned kNumberLayersForPinnedTracks = 5;
constexpr unsigned kSliderEpsilon = 4;

Reply:
Changed!